content/blog/announcing-hpe-swarm-learning-2-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing HPE Swarm Learning 2.0.0
+date: 2023-05-11T04:47:46.996Z
+featuredBlog: true
+author: HPE Swarm Learning Team
+authorimage: /img/Avatar1.svg
+disable: false
+---
+<!--StartFragment-->
+
+We’re excited to announce HPE Swarm Learning 2.0.0 community release!!
+
+
+In the previous Swarm version, if the sentinel SN goes down during Swarm training, the training process would stop, and there was no way to resume it. However, with this release, we have addressed the issue by implementing a mesh topology(connectivity) between SNs, replacing the previous star topology where only the sentinel SN was connected to other SNs. Also, we now support multiple blockchain miners instead of just one miner in the sentinel SN. Now, even if the initial sentinel SN goes down, since other SNs also function as miners, it allows the training to continue uninterrupted. Additionally, when the initial sentinel SN is down and if a new SN wants to join the network, it can seamlessly integrate and join the Swarm network with the help of any other SN node. This **high availability configuration** ensures improved resilience and robustness of  Swarm Learning.
```

Comment:
Suggest to spell "SN" and specify Swarm Learning (instead of just Swarm):

Not sure what SN means: Swarm Node? Swarm Network?
In the previous Swarm Learning version, if the sentinel Swarm Node (or Swarm Network) (SN) goes down during Swarm Learning training, the training process...

Reply:
Corrected.